admin/class-meta-columns.php

```diff
@@ -759,4 +757,64 @@ public function can_display_filter() {
 
 		return WPSEO_Post_Type::is_post_type_accessible( $screen->post_type );
 	}
+
+	/**
+	 * Sets the objects to use for the count.
+	 */
+	protected function set_indexables() {
```

Comment:
If you change the name of the global variable (`$indexables`), this should be named differently as well. Along with the `get_indexable`.